File: backend/src/main/java/com/kauffman/management/numberplate/entity/payload/NumberplateUpdatePayload.java
package com.kauffman.management.numberplate.entity.payload;

import com.kauffman.management.site.entity.dto.Site;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;
import java.util.UUID;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class NumberplateUpdatePayload {
    private UUID numberplate_id;
    private String num_plate;
    private Date dop;
    private Boolean active;
    private Site site;
}

File: frontend/src/app/private/numberplate/model/payload/numberplate-update.payload.ts
import {Site} from "../../../site/model/site";
import {PayloadInterface} from "../../../../shared/model";

export interface NumberplateUpdatePayload extends PayloadInterface  {
  site_id: string;
  num_plate: string;
  dop: Date;
  active: string;
  site: Site;
}
